db/migrate/20170227165349_add_image_file_to_facilities.rb

```diff
@@ -0,0 +1,5 @@
+class AddImageFileToFacilities < ActiveRecord::Migration
+  def change
+    add_column :facilities, :image_file, :string
```

Comment:
oh ok. I thought facilities had already an image field and we wanted also an icon